corehq/messaging/scheduling/models/content.py

```diff
@@ -0,0 +1,45 @@
+import jsonfield
+from corehq.messaging.scheduling.models.abstract import Content
+from django.db import models
+
+
+class SMSContent(Content):
+    message = jsonfield.JSONField(default=dict)
+
+    def send(self, recipient):
+        print '*******************************'
+        print 'To:', recipient
+        print 'Message: ', self.message
+        print '*******************************'
+
+
+class EmailContent(Content):
+    subject = jsonfield.JSONField(default=dict)
+    message = jsonfield.JSONField(default=dict)
+
+    def send(self, recipient):
+        print '*******************************'
+        print 'To:', recipient
+        print 'Subject: ', self.subject
+        print 'Message: ', self.message
+        print '*******************************'
+
+
+class SMSSurveyContent(Content):
+    form_unique_id = models.CharField(max_length=126)
```

Comment:
actually this points to this: https://github.com/dimagi/commcare-hq/blob/bceffe6caa457bf41c67e9524a41ccb0a8ad0cd6/corehq/apps/app_manager/models.py#L881